File: MlbDraft.API/Models/MlbTeamModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;

namespace MLBDraft.API.Models
{
    public class MlbTeamModel 
    {
     
        public Guid Id { get; set; }

        public string Description { get; set; }
        
        public string Abbreviation {get; set; }

        public string LogoPath {get; set;}

    }
}

File: MlbDraft.Client/MlbDraft/src/app/shared/models/mlbTeam.model.ts
import {Guid } from 'guid-typescript';

export class MlbTeam {
    public id: Guid;
    public description: string;
    public abbreviation: string;
    public logoPath: string;

    constructor(id: Guid, desc: string, abbrev: string, logoPath: string) {
        this.id = id;
        this.description = desc;
        this.abbreviation = abbrev;
        this.logoPath = logoPath;
    
      }
}